tests/tracer/test_trace_utils.py

```diff
@@ -123,3 +123,22 @@ def test_set_http_meta(span, int_config, method, url, status_code, query, reques
         for header, value in request_headers.items():
             tag = "http.request.headers." + header
             assert span.get_tag(tag) == value
+
+
+@pytest.mark.parametrize(
+    "error_codes,status_code,error",
+    [
+        ("404-400", 400, 1),
+        ("400-404", 400, 1),
+        ("400-404", 500, 0),
+        ("500-520", 530, 0),
+        ("500-550         ", 530, 1),
```

Comment:
```suggestion
        ("500-550", 530, 1),
```

Reply:
This test case was meant to have white space :) to make sure that any strings with excess white space is still parsed correctly

Reply:
Gotcha! I kinda figured, but wanted to be sure 🙂